Apps/Sandcastle/gallery/CZML Billboard and Label.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
```

Comment:
Can we update `CZML Billboard and Label.jpg` so the label is not missing the "A" in "AGI?"  Just use Firefox to take the screenshot.